pkgs/development/python-modules/docker-py/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, six, requests, websocket_client, docker_pycreds }:
+
+buildPythonPackage rec {
+  version = "1.10.6";
+  pname = "docker-py";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "05f49f6hnl7npmi7kigg0ibqk8s3fhzx1ivvz1kqvlv4ay3paajc";
+  };
+
+  # The tests access the network.
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    six
+    requests
+    websocket_client
+    docker_pycreds
+  ];
+
+  meta = {
+    description = "Python library for the Docker Remote API.";
```

```suggestion
    description = "Python library for the Docker Remote API";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.